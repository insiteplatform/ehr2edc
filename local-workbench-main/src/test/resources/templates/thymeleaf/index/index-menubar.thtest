%TEMPLATE_MODE HTML5
# ------------------------------------------
%CONTEXT
sections = {@eu.ehr4cr.workbench.local.controllers.NavigationItem@trialDesign,\
			@eu.ehr4cr.workbench.local.controllers.NavigationItem@cohort,\
			@eu.ehr4cr.workbench.local.controllers.NavigationItem@clinical}
section = sections[0]
user = #{ 'username' : 'John Doe' }
# ------------------------------------------
%INPUT (classpath:templates/thymeleaf/index.html)
%FRAGMENT div.header
# ------------------------------------------
%OUTPUT (classpath:templates/thymeleaf/index/expected/index-menubar.html)
