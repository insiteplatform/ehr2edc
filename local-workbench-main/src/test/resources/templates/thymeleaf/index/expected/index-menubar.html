<div class="header grid-content shrink padding-none">
    <div class="header-content menu-group primary">
        <div class="menu-group left">
            <ul class="menu-bar condense">
                <li class="logo">
                    <a href="/testing/home">
                        <img src="/testing/assets/index/img/insite-trinetx-clinical-workbench-logo-white.svg" alt="" style="border: 0;"/>
                    </a>
                </li>
                <li class="is-active">
                    <a href="/testing/app/trialDesign">InSite Studies</a>
                </li>
            </ul>
        </div>
        <div class="menu-group right">
            <ul class="menu-bar condense">
                <li class="user-menu">
                    <a href="#" class="menu-right__trigger">
                        <span>John Doe</span><i class="fa fa-angle-down"></i>
                    </a>
                </li>
                <li class="notifications-menu"><a class="notifications"><i class="fa fa-bell"></i> Notifications</a>
                </li>
            </ul>
        </div>
    </div>
</div>